docs/src/catalyst_applications/simulation_structure_interfacing.md

```diff
@@ -52,82 +108,77 @@ oprob2.tspan
 ```@example ex1
 oprob2.p
 ```
-Please note that, currently, `remake` does not work while giving `Symbol`s as input (e.g `[:X1 => 10.0, :X2 => 50.0]`), but we need to unpack the symbolic variables and use them instead (please see the end of this tutorial for more information on using symbolic variables rather than `Symbol`s).
 
 When using `remake`, we only have to provide the fields that we actually wish to change, e.g.
 ```@example ex1
-oprob3 = remake(oprob1; u0=[X1 => 10.0, X2 => 50.0])
+oprob3 = remake(oprob; u0 = [X1 => 10.0])
 nothing    # hide
 ```
-will only update the initial conditions.
-
+will only update `X1`'s value in the initial conditions.
 
-## Interfacing integrator objects
-
-During a simulation, the solution is stored in an integrator object, we will here describe how to interface with these. The primary circumstance under which a user may wish to do so is when using [callbacks](@ref advanced_simulations_callbacks). We can create an integrator by calling `init` on our problem ([while circumstances where the user might want to use `init` function exist](https://docs.sciml.ai/DiffEqDocs/stable/basics/integrator/#Initialization-and-Stepping), since integrators are automatically created during simulations, these are rare):
```

Comment:
Git got messed up here. This is marked as deleted, which it is not. Can we salvage this, should we ignore it, or do we start from the beginning on a new PR once the other stuff is merged?